### PR TITLE
Feat/new user redirect

### PR DESCRIPTION
**Pull request by spierceVR**
re #49 

Redirects new users to their profile page
Accomplishes this using cookies-next to check if profile page has been visited already.

Also changes the text color on the login page to be actually readable.

### CODE REVIEW

File: apps/portal/pages/index.tsx [hunk 1]
```diff
@@ -6,7 +6,12 @@ import { ACMButton } from '@acmutd/acm-ui';
 // commit with above line active and below line uncommented, use below line for testing if you want
 // import { ACMButton } from '@acmutd/acm-ui/src';
 
-export default function HomePage() {
+export const getServerSideProps = async (ctx) => {
+  const { profileVisited } = ctx.req.cookies;
+  return { props: { profileVisited: profileVisited ?? null } };
```
Comment (NickBurnett) [inline apps/portal/pages/index.tsx:9]: Couldn't this just be `return { props: { profileVisited } };`?

File: apps/portal/pages/profile/index.tsx [hunk 2]
```diff
@@ -5,18 +5,31 @@
...
 import ProfileField from 'components/ProfileField';
 import { Profile } from '@prisma/client';
 import { useSession } from 'next-auth/react';
 import { useForm } from 'react-hook-form';
 import Router from 'next/router';
 
-export default function ProfilePage() {
+export const getServerSideProps = async (ctx) => {
+  const { profileVisited } = ctx.req.cookies;
+  return { props: { profileVisited: profileVisited ?? null } };
```
Comment (NickBurnett) [inline apps/portal/pages/profile/index.tsx:21]: Same question.

**Review by NickBurnett — DISMISSED** [review]
Non-blocking comments.

File: apps/portal/pages/index.tsx (hunk 1 above)
Comment (spierceVR) [reply apps/portal/pages/index.tsx:9]: > Couldn't this just be `return { props: { profileVisited } };`?

image

No, apparently it does need to be like this

**Review by DedsecKnight — APPROVED** [review]

**Pull request merged**